sda-commons-starter-web/src/test/java/org/sdase/commons/spring/boot/web/security/test/AdminController.java

```diff
@@ -7,14 +7,15 @@
  */
 package org.sdase.commons.spring.boot.web.security.test;
 
-import org.springframework.boot.actuate.endpoint.web.annotation.RestControllerEndpoint;
+import org.springframework.boot.actuate.endpoint.annotation.Endpoint;
+import org.springframework.boot.actuate.endpoint.annotation.ReadOperation;
 import org.springframework.stereotype.Component;
-import org.springframework.web.bind.annotation.GetMapping;
 
+@Endpoint(id = "tasks")
```

Comment:
Is the old variant with `@RestControllerEndpoint` and `@GetMapping` deprecated or removed?

If it's removed, we should consider making the release a new major.

Reply:
RestControllerEndpoint  was just  deprecated